README.md

```diff
@@ -8,13 +8,13 @@
 
 [![dockeri.co](https://dockeri.co/image/zixia/simple-mail-forwarder)](https://hub.docker.com/r/zixia/simple-mail-forwarder/)
 
-Simplest and Smallest Email Forward Service based on Docker.
+Simplest and Smallest Email Forward Service for Docker.
 
-1. Config by [**one-line**](#environment-variable-and-default)
+<!-- @Deprecated 1. Config by [**one-line**](#environment-variable-and-default) -->
```

Comment:
Cloud you please let me know why you suggest deprecate this line?

I believe this is still a very important feature for SMF.

Reply:
the reference link no longer works, the headline #environment-variable-and-default doesn't exist and i couldn't find where the "new" version is

Reply:
`#environment-variable-and-default-values`

I believe the reference link should be corrected to this: <https://github.com/huan/docker-simple-mail-forwarder#environment-variable-and-default-values>